content/docs/studio/view-settings.md

```diff
@@ -0,0 +1,92 @@
+# Additional Settings for a View
+
+If you are creating a view for a DVC repo, and if the DVC repo is at the root of
+the Git repository and does not reference remote/cloud storage, then you can
+successfully visualize it without specifying additional settings.
+
+In other cases, you could be trying to create views from non-DVC repositories.
+Or, your project may be in a sub-directory inside a monorepo. Or, the data to be
+visualized could be in custom files in your repository or even in remote / cloud
+storages. In all these scenarios, you will need to specify advanced settings so
+that DVC Studio is able to access the data required for visualization. These
+scenarios have been explained in details below.
+
+## Non-DVC repositories
+
+DVC Studio creates views by identifying datasets, metrics and hyperparameters
+defined in your Git repositories. These details (datasets, metrics and
+hyperparameters) are stored in your Git repositories as CSV, JSON or YAML files.
+You can add these details to your Git repositories in two ways:
+
+1. **Set up DVC repositories**: You can use [DVC](https://dvc.org/) and Git to
+   version your code, data and models all within your Git repositories. By using
+   DVC, you can be sure not to bloat your repositories with large volumes of
+   data or huge models. These large assets reside in cloud or other remote
+   storage locations. You will simply track their version info in Git.
```

Comment:
do we need to explain what DVC is over here? In which case there's probably already some other paragraph from elsewhere in the docs which we can copy-paste or link to.

Reply:
Yes. Will add.

Reply:
Added to #2515